crates/oro-client/src/client.rs

```diff
@@ -44,17 +88,47 @@ impl OroClientBuilder {
         self
     }
 
+    #[cfg(not(target_arch = "wasm32"))]
+    pub fn set_proxy(mut self, proxy: bool) -> Self {
+        self.proxy_config.proxy = proxy;
+        self
+    }
+
+    #[cfg(not(target_arch = "wasm32"))]
+    pub fn set_proxy_url(mut self, proxy_url: impl AsRef<str>) -> Self {
+        self.proxy_config.proxy_url = Some(proxy_url.as_ref().into());
```

Comment:
You should create the `Proxy` object here, and have _this_ return `Result<Self, OroClientError>`, and error out on Url parse errors or `Proxy` creation errors, so we can skip unwrapping in `.build()`